### PR TITLE
Feat/address copy

### PR DESCRIPTION
**Pull request by divyam1705**
- adds the ability to copy the account address

### CODE REVIEW

File: packages/frontend/src/components/connect-button/account-menu/index.tsx [hunk 1]
```diff
@@ -46,6 +48,14 @@ export function AccountMenu({
     const rootRef = useRef(null);
     const { disconnect } = useDisconnect();
 
+    const [copied, setCopied] = useState(false);
+
+    const handleCopyClick = () => {
+        navigator.clipboard.writeText(account.address).then(() => {
```
Comment (luzzif) [inline packages/frontend/src/components/connect-button/account-menu/index.tsx:54]: From `caniuse` I see that this has roughly 93% of users support. I'm wondering if we should use a library to guarantee better compatibility or at least make this a bit more sophisticated.

**Review by luzzif — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/connect-button/account-menu/styles.module.css [hunk 2]
```diff
@@ -11,7 +11,11 @@
 }
 
 .accountContainer {
-    @apply flex items-center gap-3;
+    @apply flex items-center gap-3 hover:bg-gray-200 p-2 rounded-md cursor-pointer;
```
Comment (guerrap) [inline packages/frontend/src/components/connect-button/account-menu/styles.module.css:14]: We should add the usual `duration-200 transition-colors ease-in-out` when we change styles on user interactions.

**Comment by 0xVenky** [comment]
hey, looks good. I am wondering if we should extend the duration. It feels the copied state goes away immediately. Maybe keep it for 1.5 to 2 seconds?

File: packages/frontend/src/components/connect-button/account-menu/styles.module.css [hunk 3]
```diff
@@ -11,7 +11,19 @@
...
 .accountContainer {
-    @apply flex items-center gap-3 hover:bg-gray-200 p-2 rounded-md cursor-pointer;
+    @apply flex 
+        items-center 
+        gap-3 
+        hover:bg-gray-200 
+        p-2 
+        rounded-md 
+        cursor-pointer 
+        transition-[outline]
```
Comment (guerrap) [inline packages/frontend/src/components/connect-button/account-menu/styles.module.css:21]: Let's have a transition for the background too.

**Review by guerrap — APPROVED** [review]

**Review by luzzif — APPROVED** [review]

**Pull request merged**